velox/docs/functions/spark/string.rst

```diff
@@ -427,3 +427,25 @@ String Functions
     Returns string with all characters changed to uppercase. ::
 
         SELECT upper('SparkSql'); -- SPARKSQL
+
+.. spark:function:: varchar_type_write_side_check(string, limit) -> varchar
+
+    Removes trailing space characters (ASCII 32) that exceed the maximum Unicode string length ``limit`` in characters.
+    This function will trim at most the length of ``string`` in characters - ``limit`` space characters from the end of this string.
+    Throws exception when the string still exceeds ``limit`` after trimming trailing spaces or when ``limit`` is negative.
+    Note: This function is not directly callable in SQL, but internally used by SparkSQL for length check when writing to
```

Comment:
nit: in SQL -> in Spark SQL

> internally used by SparkSQL for length check when writing varchar type columns

Comment suggestion:

internally used for length check when wring string type columns